### PR TITLE
xar: add page

### PR DESCRIPTION
**Pull request by johan456789**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/osx/xar.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# xar
+
+> Manage .xar archives.
+
```
Comment (sbrl) [inline pages/osx/xar.md]: Perhaps we could insert a line here that says this?

```markdown
> Homepage: <https://mackyle.github.io/xar/>.

File: pages/osx/xar.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- Create a xar archive of all files in a given directory:
+
+`xar -cf {{archive.xar}} {{path/to/directory}}`
+
+- List the contents of a given xar archive:
+
+`xar -tf {{archive.xar}}`
```
Comment (agnivade) [inline pages/osx/xar.md]: Let's use `--list` instead of `-t`

Comment (johan456789) [reply pages/osx/xar.md]: I tried but `--list` doesn't work. There's also no such usage [here](https://linux.die.net/man/1/xar).

Comment (agnivade) [reply pages/osx/xar.md]: This page https://mackyle.github.io/xar/xar.html mentions that `--list` is a synonym for `-t`. Is it a version related issue ? Or the doc is incorrect ?

Comment (pxgamer) [reply pages/osx/xar.md]: It looks like it was originally in subversion (the old version) and then forked to GitHub and updated.

Comment (agnivade) [reply pages/osx/xar.md]: Even the github page has the `--list` option - https://github.com/mackyle/xar/blob/master/xar/src/xar.1#L29

Comment (pxgamer) [reply pages/osx/xar.md]: Yeah, the new versions do. The man page on Linux.die is the original older version.

Comment (johan456789) [reply pages/osx/xar.md]:
```
$ which xar
/usr/bin/xar
```
Does this mean it's preinstalled on mac?

Comment (agnivade) [reply pages/osx/xar.md]: Not necessarily. Any package is free to install a command at `/usr/bin`. Is there a brew version of xar that you can get the latest version of ?

Comment (johan456789) [reply pages/osx/xar.md]:
```
$ brew search xar
==> Formulae
xar-mackyle                              xaric
$ brew info xar-mackyle
xar-mackyle: stable 1.6.1 (bottled)
eXtensible ARchiver
https://mackyle.github.io/xar/
Not installed
```
I just checked my friend's mac. It also has `xar`.  I'm guessing `xar` comes preinstalled on mac and the github version of `xar` is different from the one on mac.
Also, mac version:
```
$ xar --version
xar 1.8dev
```

Comment (agnivade) [reply pages/osx/xar.md]: Ok, so the mac version is more recent than the brew version ? And that still doesn't have the --list option ?

Comment (johan456789) [reply pages/osx/xar.md]: It seems so. Should I remove the GitHub link as homepage?

Comment (agnivade) [reply pages/osx/xar.md]: Sure. Then put this https://linux.die.net/man/1/xar as the home page. And change `list` to `lis[t]`

Comment (johan456789) [reply pages/osx/xar.md]: Sure, changed it

**Pull request merged**